manifests/pipeline.yaml

```diff
@@ -35,7 +35,13 @@ parameters:
     value: quay.io/coreos-assembler/coreos-assembler:main
   - description: AWS S3 bucket in which to store builds (or blank for none)
     name: S3_BUCKET
-    value: fcos-builds
+    value: ''
```

Comment:
Was this intended?

Reply:
Yes, you have to submit it.  The S3 bucket will be different for fcos/rhcos